src/ResourceManager/Network/Commands.Network/ServiceEndpointPolicy/GetAzureServiceEndpointPolicyCommand.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Commands.Network
     using System.Collections.Generic;
     using System.Management.Automation;
 
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ServiceEndpointPolicy", SupportsShouldProcess = true, DefaultParameterSetName = "GetByNameParameterSet"), OutputType(typeof(PSServiceEndpointPolicy))]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ServiceEndpointPolicy", SupportsShouldProcess = true, DefaultParameterSetName = "ListParameterSet"), OutputType(typeof(PSServiceEndpointPolicy))]
```

Comment:
why are you changing the default behavior?

Reply:
I updated cmdlet to cover all the swagger Get/List/ListAll. So I had to update default parameter set to avoid prompting for RG and Name in case they were not specified (i.e. List All operation ).
This should'n break existing scenarios as all the parameters combination are kept and should be processed as before.